### PR TITLE
app-misc/rox-filer: Fix passing of incompatible pointer type

### PR DESCRIPTION
**Pull request by listout**
And update EAPI 7 -> 8

Closes: https://bugs.gentoo.org/924712

### CODE REVIEW

**Comment by listout** [comment]
Again libglade dependency.

**Comment by ppurka** [comment]
duplicate of https://github.com/gentoo/gentoo/pull/35407

File: app-misc/rox-filer/rox-filer-2.11-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	x11-libs/gtk+:2
+	x11-libs/libSM"
+RDEPEND="${COMMON_DEPEND}
+	x11-misc/shared-mime-info"
+DEPEND="${COMMON_DEPEND}
+	dev-util/intltool
+	sys-devel/gettext"
+BDEPEND="virtual/pkgconfig"
+
+S="${WORKDIR}/${P}/ROX-Filer/src"
```
Comment (mm1ke) [inline app-misc/rox-filer/rox-filer-2.11-r1.ebuild:28]: can you move this below `SRC_URI` ?

**Comment by juippis** [comment]
Let's find a resolution in #35407

**Pull request closed without merging**